python/pacemaker/_cts/corosync.py

```diff
@@ -11,6 +11,8 @@
 from pacemaker.buildoptions import BuildOptions
 from pacemaker._cts.process import killall, stdout_from_command
 
+corosync_cfg_file = BuildOptions.COROSYNC_CONFIG_DIR + "/corosync.conf"
```

Comment:
Bah, this is actually a bug in configure.ac :-/ It is supposed to be the file path. It looks like the only use of the variable is in cts/lab/CIB.py, and it's probably broken when corosync provides pkg-config variables. configure.ac correctly uses the file path in all other cases.

Reply:
Fixed.